fastexcel-writer/src/main/java/org/dhatim/fastexcel/Range.java

```diff
@@ -216,13 +233,32 @@ public CustomDataValidation validateWithFormula(String formula) {
     /**
      * Specifically define this range by assigning it a name.
      * It will be visible in the cell range dropdown menu.
-     * 
+     *
      * @param name string representing the name of this cell range
      */
     public void setName(String name) {
         worksheet.addNamedRange(this, name);
     }
 
+    /**
+     * Return the visibility of this range.
+     *
+     * @return {@code true} if the range is visible by all worksheet, {@code false} if it is visible only by the worksheet contains the range
+     */
+    public boolean isGlobalVisibility() {
+        return globalVisibility;
+    }
+
+    /**
+     * Set the visibility of this range
+     *
+     * @param globalVisibility {@code true} to allow to see the range by all worksheet
+     */
+    public Range setGlobalVisibility(boolean globalVisibility) {
+        this.globalVisibility = globalVisibility;
+        return this;
```

Comment:
other setters return void. returning this here seems inconsistent

Reply:
I understand. I change it